src/installer.js

```diff
@@ -152,15 +152,18 @@ function getOptions (data, defaults) {
   if (options.description) {
     // Replace all newlines in the description with spaces, since it's supposed
     // to be one line.
-    options.description = options.description.replace(/[\r\n]+/g, ' ')
+    options.description = options.description.replace(/[\n]+/g, ' ')
```

Comment:
Why remove `\r`? Wouldn't `package.json` files saved on Windows be problematic?

Reply:
Since we removed support for Windows when adding the `os` option to our `package.json` people probably won't be using this on Windows. The change will re-enforce this.
I guess there's a chance that somebody uses mainly Windows to develop a `deb` package and move files across OSs constantly, and decides to use CR+LF as their main new-line character

Reply:
Yeah, they could develop on a Windows machine and then use Travis CI (or similar) to build their distributables.

Reply:
True! I'll revert it to what it was, and also change it on the `productDescription` side